portal/get_patient_info.php

```diff
@@ -16,164 +16,182 @@
  * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
  *
  * @package OpenEMR
- * @author Jerry Padgett <sjpadgett@gmail.com>
```

Comment:
Why did you remove the other Jerry Padgett?
(sorry, couldn't help myself; it's still too early in the day for me)